packages/lpa-questionnaire-web-app/src/controllers/contact-us.js

```diff
@@ -0,0 +1,13 @@
+const { VIEW } = require('../lib/views');
+const { renderView } = require('../util/render');
+
+async function renderContactUs(req, res) {
```

Comment:
Generally the controller actions are prefixed with `get` or `post` so it would be better if this action followed that for consistency, for example `getContactUs`.